### DISCUSSION TITLE
Updating from 10.5 to 11, migration error when rebuilding ObjectBrick

### DISCUSSION THREAD

**Discussion by julescournut** [pimcore/pimcore#15857] [Q&A / Support] [answered]
Hello,

I'm currently updating a Pimcore from 10.5 to 11.0 follwing this documentation page https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/Updating_Pimcore/V10_to_V11#rebuild-classes-objectbricks-fieldcollections-and-customlayouts

I have an error when I execute this migration Pimcore\Bundle\CoreBundle\Migrations\Version20230412105530

```
Pimcore\Model\DataObject\ClassDefinition\Data\ManyToManyObjectRelation::setMaxItems(): Argument #1 ($maxItems) mus  
  t be of type ?int, string given, called in /var/www/html/vendor/pimcore/pimcore/models/DataObject/ClassDefinition/  
  Data.php on line 227
```

My ObjectBricks are completely empty in Pimcore interface.

Thanks for your help.

**Comment by blankse** [comment, marked as answer]
You shouldn't migrate from 10.5 direct to 11.0. You should first update to the latest 10.6 release. So the types of the Data attributes should be converted correctly.

See: https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/Updating_Pimcore/Preparing_for_V11#upgrade-to-pimcore-106

**Reply by julescournut** [reply]
Thank you for your super quick answer, it solved this problem. I have now an issue on datahub configurations but I'm still trying to figure things out, it's probably another documentation page that I missed..
Thanks again !
